hydra-node/test/Hydra/Chain/Direct/Contract/CollectCom.hs

```diff
@@ -144,7 +144,7 @@ data HealthyCommit = HealthyCommit
 
 healthyCommitOutput ::
   Party ->
-  (TxIn, TxOut CtxUTxO) ->
+  UTxO ->
```

Comment:
should this be a list?

Reply:
UTxO may already contain multiple entries. What name would be more intuitive? `UTxOSet` or `UTxOMap`?

Reply:
UTxO is fine!